src/Trakx.Kaiko.ApiClient.Stream/Clients/MarketUpdateClient.cs

```diff
@@ -139,11 +140,16 @@ private static string DefineExchanges(string[] exchanges)
             BaseSymbol = codeParts[0],
             QuoteSymbol = codeParts[1],
             Exchange = current.Exchange,
+
             Price = (decimal)current.Price,
+            Amount = (decimal)current.Amount,
```

Comment:
The definition of `Convert.ToDecimal()`:
```csharp
public static decimal ToDecimal(double value)
{
    return (decimal)value;
}
```

It's the same as the current code 😅

Reply:
Hmm, I agree with @goneneren to apply the culture in the conversion to avoid culture "surprises" 😅 
Example: `Convert.ToDecimal(current.Amount, CultureInfo.InvariantCulture);`

Reply:
Sorry, didn't noticed that wasn't a string to support encoding. My bad 😅

Reply:
All good, thank you both. We are doing a simple double to decimal conversion, which the framework itself handles with a cast, so the code in the PR seems correct.